game/configuration.cc

```diff
@@ -352,6 +392,21 @@ void readConfigXML(fs::path const& file, int mode) {
 	}
 }
 
+int PaHostApiNameToHostApiTypeId (const std::string& name) {
+	if (name == "Auto") return 1337;
+	if (name == "Windows DirectSound") return 1;
+	if (name == "MME") return 2;			
+	if (name == "ASIO") return 3;
+	if (name == "Core Audio" || name == "CoreAudio") return 5;
+	if (name == "OSS") return 7; // Not an error, stupid PortAudio.
+	if (name == "ALSA") return 8;
```

Comment:
I just tested it on my ubuntu VM and it appears to work fine.
The default device wasn't pulse on my machine but it appeared to work with all possible valid output devices (on a VM, inside a hackintosh using HDMI... that's about as fringe as fringe-cases can get)

Still, portaudio itself has no pulseaudio api, so what happens is we use ALSA and ALSA uses PulseAudio.